src/index.ts

```diff
@@ -33,17 +33,23 @@ const PKG = require('../../package.json');
 const v1 = require('./v1');
 
 import {Snapshot} from './snapshot';
-import {Subscription, SubscriptionMetadata, SubscriptionMetadataRaw} from './subscription';
+import {Subscription, SubscriptionMetadataRaw} from './subscription';
 import {Topic, PublishOptions} from './topic';
 import {CallOptions} from 'google-gax';
 import {Readable} from 'stream';
 import {google} from '../proto/pubsub';
 import {ServiceError} from 'grpc';
+import {FlowControlOptions} from './lease-manager';
 
 const opts = {} as gax.GrpcClientOptions;
 const {grpc} = new gax.GrpcClient(opts);
 
 
+export interface SnapshotParent extends PubSub {
```

Comment:
I don't think we should export something like this, a Snapshot's parent can either be a Subscription or the PubSub instance, we don't have any kind of class that mixes the two.

Reply:
Removed, i missed while merging, Thanks 👍